try.html

```diff
@@ -14,6 +14,12 @@
     Click the boxes below to learn how they work and to learn more.
     If you like one, you can find <a href="/install">installation instructions here</a>.
 
+    ⚠️Experimental⚠️ several of the environments below use the
+    <a href="https://jupyterlite.readthedocs.io/en/latest/">JupyterLite project</a> to provide a self-contained
+    Jupyter environment that runs in your browser. This is experimental technology and
+    may have some bugs, so please be patient and report any unexpected behavior in
+    <a href="https://github.com/jupyterlite/jupyterlite/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc">the JupyterLite repository</a>.
```

Comment:
```suggestion
    <br><br>
    ⚠️<strong>Experimental</strong>⚠️ several of the environments below use the
    <a href="https://jupyterlite.readthedocs.io/en/latest/">JupyterLite project</a> to provide a self-contained
    Jupyter environment that runs in your browser. This is experimental technology and
    may have some bugs, so please be patient and report any unexpected behavior in
    <a href="https://github.com/jupyterlite/jupyterlite/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc">the JupyterLite repository</a>.
```

Reply:
This breaks up the paragraph and bolds the *experimental* which I think is a cleaner look